test/test_jit.py

```diff
@@ -7320,6 +7320,14 @@ def test_call_python(a):
 
             self.assertEqual(cu.test_call_python(*inputs), outputs)
 
+    def test_type_call_in_script(self):
+        @torch.jit.script
+        def fn(x):
+            return type(x)
+
+        with self.assertRaisesRegex(RuntimeError, "value of type type"):
```

Comment:
Can you put quotes around the output here so it doesn't look like a typo?
e.g.
`value of type 'type'`